src/test/groovy/integration/TimeoutFuncSpec.groovy

```diff
@@ -28,12 +30,44 @@ class TimeoutFuncSpec extends IntegrationSpec {
             projectName << ['project_with_timeout', 'project_with_timeout_with_spock_1']
     }
 
+    @Unroll
+    def "should not fail the tests in project [#projectName] due to timeout because of regexps"() {
+        given:
+            copyResources(projectName, "")
+        when:
+            ExecutionResult result = runTasksWithFailure("build", "profileTests")
```

Comment:
You check if the whole build failed. Maybe it would be good to add a test case to not break the build when only excluded/ignored tests were timed out?